torch/nn/utils/rnn.py

```diff
@@ -270,8 +270,11 @@ def pack_sequence(sequences):
 
     Arguments:
         sequences (list[Tensor]): A list of sequences of decreasing length.
+        batch_first (bool, optional): if ``True``, the output will be in ``B x T x *``
+            format.
```

Comment:
That's not really true. The output is a `PackedSequence`, so it's not a tensor that's this easy to describe

Reply:
Yeah "internal" batch first format, how would you best describe it ?